### ISSUE TITLE
 skin detachment

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#3650]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

skin detachment

### 2. Synonyms

detached skin; skin sloughing

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

loss of sections of skin either spontaneously or after gentle handling (definition in MP) PMID:19018717

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

Abnormality of skin morphology (HP:0011121)

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

Stevens–Johnson syndrome; Toxic Epidermal Necrolysis (OMIM:608579)

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))

RGD:gthayman

**Comment by pnrobinson** [comment]
https://hpo.jax.org/app/browse/term/HP:0032156
